assets/home/main.js

```diff
@@ -58,9 +58,7 @@ const Main = () => {
 
 				{ topRow }
 
-				<Col as="section" className="sensei-home__section" cols={ 12 }>
```

Comment:
I moved the `Col` to the `TasksSection` component, so we can keep all the logic related to the Tasks there, like not rendering the whole `<Col>` when it was dismissed.